test/handler/test_dart_analyze_handler.vader

```diff
@@ -9,20 +9,20 @@ Execute(Basic problems should be parsed correctly):
   \ [
   \   {
   \     'type': 'E',
-  \     'text': 'expected_token: Expected to find ''}''',
-  \     'lnum': 5,
-  \     'col': 1,
+  \     'text': 'invalid_assignment: A value of type ''String'' can''t be assigned to a variable of type ''int''. Try changing the type of the variable, or casting the right-hand type to ''int''.',
+  \     'lnum': 7,
+  \     'col': 10,
   \   },
   \   {
-  \     'type': 'W',
-  \     'text': 'invalid_assignment: A value of type ''String'' can''t be assigned to a variable of type ''int''',
-  \     'lnum': 2,
-  \     'col': 16,
+  \     'type': 'I',
+  \     'text': 'prefer_const_constructors: Prefer const with constant constructors.',
+  \     'lnum': 9,
+  \     'col': 10,
   \   },
   \ ],
   \ ale_linters#dart#dart_analyze#Handle(bufnr(''), [
   \ 'Analyzing main.dart...',
-  \ '  error - main.dart:5:1 - Expected to find ''}'' - expected_token',
```

Comment:
I think these changes should not affect the already exising tests. Please keep the previous tests and add the new ones.